orchagent/dash/dashaclorch.cpp

```diff
@@ -285,16 +285,16 @@ sai_attr_id_t getSaiStage(DashAclDirection d, sai_ip_addr_family_t f, DashAclSta
     return stage->second;
 }
 
-DashAclOrch::DashAclOrch(DBConnector *db, const vector<string> &tables, DashOrch *dash_orch) :
-    Orch(db, tables),
+DashAclOrch::DashAclOrch(DBConnector *db, const vector<string> &tables, DashOrch *dash_orch, ZmqServer *zmqServer) :
+    ZmqOrch(db, tables, zmqServer),
     m_dash_orch(dash_orch)
 {
     SWSS_LOG_ENTER();
 
     assert(m_dash_orch);
 }
 
-void DashAclOrch::doTask(Consumer &consumer)
+void DashAclOrch::doTask(ZmqConsumer &consumer)
```

Comment:
>ZmqConsumer [](http://example.com/codeflow?start=25&length=11)

Possible to use ConsumerBase?

Reply:
No, because DashAclOrch inherite from ZmqOrch, this method override a virtual method from ZmqOrch::doTask(ZmqConsumer &consumer).

We can't define ZmqOrch::doTask(ConsumerBase &consumer). because the doTask code implementation for Consumer  and ZmqConsumer  are different.